schema/taxonomy/ecs/source.yaml

```diff
@@ -1,7 +1,8 @@
+# This is an example taxonomy to demonstrate how one could go after mapping
+# different concrete tools to ECS.
```

Comment:
As long as we don't recursivly taxonomies from subfolders, this won't get picked up. Therefore the failing integration test. Let's make this change in a later PR.

Reply:
I've added a story for this.